src/features/elementfeature.jl

```diff
@@ -91,24 +95,16 @@ end
 
 encodable_elements(efd::ElementFeatureDescriptor) = efd.lookup_table[:, :Symbol]
 
-function encodable_elements(feature_name::String, lookup_table::DataFrame = atom_data_df)
-    info = lookup_table[:, [Symbol(feature_name), :Symbol]]
-    return info[
-        findall(x -> !ismissing(x), getproperty(info, Symbol(feature_name))),
-        :Symbol,
-    ]
-end
```

Comment:
This function would allow users to see what elements can be encoded with a  specific feature without actually having to create a FeatureDescriptor. Is that something we don't want to have?

Reply:
It just felt a bit inconsistent with the rest of the API to me...I'm not deadset on removing it; we could put it back. But it's also not that hard to just do `encodable_elements(ElementFeatureDescriptor(feature_name))`...:woman_shrugging:

Reply:
Fair enough. I was just curious and don't feel strongly about either option 😄